ballista/ui/scheduler/src/components/QueriesList.tsx

```diff
@@ -94,13 +94,20 @@ export const ActionsCell: (props: any) => React.ReactNode = (props: any) => {
   };
   return (
     <Flex>
-      <FaStop color={"red"} title={"Stop this job"} />
+      <FaStop color={"red"} title={"Stop this job"} onClick={() => {
+        fetch("api/job/" + props.value.job_id, {
```

Comment:
This works succesfully - but has the "infinite loop" problem
https://github.com/apache/arrow-ballista/issues/341